x-pack/plugins/security/server/routes/api/public/roles/get.test.js

```diff
@@ -87,7 +87,7 @@ describe('GET roles', () => {
       },
     });
 
-    getRolesTest(`throws error if resource isn't * and doesn't have the space: prefix`, {
+    getRolesTest(`return error if we have empty resources`, {
```

Comment:
I know we didn't directly talk about this scenario, but I would almost expect this to return a `_transform_error: ['kibana']` instead of a straight `500` response. What do you think?

Reply:
Yeah, that's the most consistent, I will make it so.

Reply:
Oh, now I remember why I didn't do so. This should never happen in Elasticsearch, and I couldn't recall why I ever made it behave this way before.